aws/efs.tf

```diff
@@ -30,6 +30,12 @@ resource "aws_efs_mount_target" "home_dirs_targets" {
   security_groups = [ aws_security_group.home_dirs_sg.id ]
 }
 
+data "helm_repository" "stable" {
```

Comment:
IIRC this block is now invalid.  Because it is invalid,  and/or url's changed,  wherever it was referenced Jake just switched to a hard coded url.   Maybe Slack Jake and delete this.  I was able to deploy in TESS DEV without it.

Reply:
Block removed